driver/test_utils.go

```diff
@@ -77,13 +59,13 @@ func newMockBackupStorage(s *upcloud.Storage) *upcloud.Storage {
 }
 
 func (m *mockUpCloudDriver) Run() error {
-	fmt.Println("sup")
+	log.Println("sup")
```

Comment:
Should this have a TODO? Not your changes of course, but feels a lot like a placeholder :)

Reply:
I think that we can remove that whole line :)